test/components/kubernetes/controlplane-multi-node_disruptive_test.go

```diff
@@ -0,0 +1,59 @@
+// Copyright 2020 The Lokomotive Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// +build aws aws_edge
+// +build disruptivee2e
+
+package kubernetes_test
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	testutil "github.com/kinvolk/lokomotive/test/components/util"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func Test_Controlplane_components_DaemonSets_can_be_gracefully_updated(t *testing.T) {
+	client := testutil.CreateKubeClient(t)
+	deployClient := client.AppsV1().DaemonSets(namespace)
```

Comment:
Copypasta? This should be daemoset or ds.

Reply:
Yes, thanks, fixed.